Microsoft.Toolkit.Uwp.SampleApp/SamplePages/CameraPreview/CameraPreviewPage.xaml

```diff
@@ -0,0 +1,17 @@
+﻿<Page
+    x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.CameraPreviewPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">    
+    <StackPanel Orientation="Vertical" Margin="20">
+        <controls:CameraPreview x:Name="CameraPreviewControl" 
+                                     VideoFrameArrived="CameraPreviewControl_VideoFrameArrived"
+                                     SoftwareBitmapArrived="CameraPreviewControl_SoftwareBitmapArrived"></controls:CameraPreview>
+        <TextBlock x:Name="ImageHeader" Text="Image" ></TextBlock>
+        <Image x:Name="CurrentFrameImage" MinWidth="300" Width="400" HorizontalAlignment="Left"></Image>
```

Comment:
isn't the goal of CameraPreview to preview the stream? why is there an image control as well in this page then?